sdk/src/lib.rs

```diff
@@ -169,6 +169,8 @@ pub use solana_sdk_macro::declare_id;
 pub use solana_sdk_macro::pubkeys;
 #[deprecated(since = "2.1.0", note = "Use `solana-secp256k1-recover` crate instead")]
 pub use solana_secp256k1_recover as secp256k1_recover;
+#[deprecated(since = "2.1.0", note = "Use `solana-secp256r1` crate instead")]
+pub use solana_secp256r1 as secp256r1_program;
```

Comment:
We don't need to add a re-export here. The other re-exports here were just to avoid breaking changes

Reply:
Removed the re-exports from `sdk/program/src/lib.rs` and `sdk/src/lib.rs`respectively, with [4d5b213](https://github.com/anza-xyz/agave/pull/3152/commits/4d5b213e4e332cde0f7d640725828c5ee577a663) ✅